lib/screens/music_screen.dart

```diff
@@ -22,6 +23,29 @@ import '../services/finamp_settings_helper.dart';
 import '../services/finamp_user_helper.dart';
 import '../services/jellyfin_api_helper.dart';
 
+void postLaunchHook(WidgetRef ref) async {
```

Comment:
Why does this function get a widgetref?

Reply:
I used a provider instead of the "raw" method from the downloads service at first before changing it, and thought that it might be useful to have access to a WidgetRef here in the future. Or is that an anti-pattern?

Reply:
I know that widgetrefs/buildcontexts are heavy and you need to be careful not to accidentally capture a one in a lambda when you don't mean to, but that isn't really relevant with a normal function.  It just seemed a bit odd to have a completely unused parameter like this.

Reply:
Yeah it's more of a nice-to-have here, not yet necessary. It's just sometimes anoying to get a WidgetRef where and when you need one, so I saw an opportunity here ^^